documentation/examples/prometheus.yml

```diff
@@ -27,4 +27,17 @@ scrape_configs:
     # scheme defaults to 'http'.
 
     static_configs:
-      - targets: ['localhost:9090']
+      - targets: 
+        - 'localhost:9090'
+
+  # An example for relabeling an instance
```

Comment:
This file is where people look.

We need to be a bit careful with example configs. Things that are generic, useful and don't point users down the wrong path will be tricky to come up with.